KMD.Identity.TestApplications.SAML.MVCCore/Controllers/AuthController.cs

```diff
@@ -30,6 +30,15 @@ public IActionResult Login(string returnUrl = null, string domainHint = null)
         {
             var binding = new Saml2RedirectBinding();
             binding.SetRelayStateQuery(new Dictionary<string, string> { { relayStateReturnUrl, returnUrl ?? Url.Content("~/") } });
+            if (HttpContext.User.Identity.IsAuthenticated)
```

Comment:
I think you should avoid multiline if-statements without curly braces around. It is a bit harder to read and more error prone if you edit it later. Also it just looks weird, since you see it so rarely :)

Reply:
done